hollow/src/main/java/com/netflix/hollow/api/producer/validation/DuplicateDataDetectionValidator.java

```diff
@@ -44,7 +45,9 @@
  * this validator will not fail as the two objects are deduped.
  *
  */
-public class DuplicateDataDetectionValidator implements Validator {
+public class DuplicateDataDetectionValidator implements Nameable, Validator {
+	private final String NAME = "DuplicateDataDetectionValidator";
```

Comment:
Small smell that we're duplicating the class name here. Maybe we can just use short class name instead of a constant string? Or change the mechanism for constructing the name mentioned below).

Reply:
Using short class name works too. The validator can choose an appropriate name to return. I just chose this but point taken.